addons/languages/coptic/pack/src/main/res/xml/coptic_swift.xml

```diff
@@ -0,0 +1,37 @@
+<Keyboard xmlns:android="http://schemas.android.com/apk/res/android" xmlns:ask="http://schemas.android.com/apk/res-auto" android:keyWidth="10.0%p" android:keyHeight="@integer/key_normal_height" ask:autoCap="false">
+  <Row>
+    <Key android:keyEdgeFlags="left" android:codes="ⲑ"/>
+    <Key android:codes="ϣ"/>
+    <Key android:codes="ⲉ"/>
+    <Key android:codes="ⲣ" android:popupCharacters="ⳁ"/>
+    <Key android:codes="ⲧ" android:popupCharacters="ϯ"/>
+    <Key android:codes="ⲯ"/>
+    <Key android:codes="ⲩ"/>
+    <Key android:codes="ⲓ"/>
+    <Key android:codes="ⲟ"/>
+    <Key android:keyEdgeFlags="right" android:codes="ⲡ"/>
+  </Row>
+  <Row>
+    <Key android:keyEdgeFlags="left" android:codes="ⲁ"/>
+    <Key android:codes="ⲥ" android:popupCharacters="ⲋ"/>
+    <Key android:codes="ⲇ" android:popupCharacters="⳧"/>
+    <Key android:codes="ϥ" android:popupCharacters="ⲫ"/>
+    <Key android:codes="ⲅ"/>
+    <Key android:codes="ⲏ" android:popupCharacters="ϩ"/>
+    <Key android:codes="ⲓ" android:popupCharacters="ϭ"/>
```

Comment:
This duplicates the iota in the upper row. Probably should be ϫ which is otherwise missing.

Reply:
I messed up there, fixed it.